### PR TITLE
Set up github actions

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -19,6 +19,8 @@ exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory/ec2.py,awx/plugins
 max-line-length=160
 ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293,E731,F405
 exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory,awx/ui,awx/api/urls.py,awx/main/migrations,awx/main/south_migrations,awx/main/tests/data,node_modules/,awx/projects/,tools/docker,awx/settings/local_*.py,installer/openshift/settings.py,build/,installer/
+per-file-ignores =
```
Comment (j00bar) [inline setup.cfg:22]: TIL you can do this.

Comment (jeremy-audet) [reply setup.cfg:22]: One can also ignore errors inline with comments:

```python
print('foo')  # noqa: E731,E123
```

That'd be more maintainable over time, as lines of code shift around. Less spooky action at a distance.

File: .github/workflows/pythonapp.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Test with pytest
+        run: |
+          pip install pytest
+          python setup.py develop
```
Comment (jeremy-audet) [inline .github/workflows/pythonapp.yml:55]: What is the purpose of this line of code?

File: setup.cfg (hunk 1 above)
Comment (adamruzicka) [reply setup.cfg:22]: There was like twenty long lines in that file and I didn't really want to ignore them one by one

File: .github/workflows/pythonapp.yml (hunk 2 above)
Comment (adamruzicka) [reply .github/workflows/pythonapp.yml:55]: It makes the plugin module available in the environment? Something like that. @matburt Care to help me here, I'm not realloy a python person

Comment (jeremy-audet) [reply .github/workflows/pythonapp.yml:55]: Ahhh, yeah, I see why this is here. It installs the code in develop/editable mode, which is needed for stuff like imports to resolve. It's just a much less common technique than `pip install -e .` or similar.

Comment (adamruzicka) [reply .github/workflows/pythonapp.yml:55]: Should I change it to use `pip`?

Comment (jeremy-audet) [reply .github/workflows/pythonapp.yml:55]: AFAIK, the end result is about the same. This seems fine.

**Comment by j00bar** [comment]
Per our conversation today, these actions and tests need to run on a py3.6 environment, not py3.7

**Pull request merged**